Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    vrt (0.4.2)
+    vrt (0.4.3)
```

Comment:
should the patch version change?

Reply:
it shouldn't, its just that the release was botched last time

Reply:
Hey! It wasn't botched.
It was "done creatively"